src/main/kotlin/fi/metatavu/metaform/server/controllers/ScriptController.kt

```diff
@@ -0,0 +1,37 @@
+package fi.metatavu.metaform.server.controllers
+
+import fi.metatavu.metaform.api.spec.model.MetaformScript
+import fi.metatavu.metaform.server.script.RunnableScript
+import fi.metatavu.metaform.server.script.ScriptProcessor
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+/**
+ * Controller for Metaform scripts
+ */
+@ApplicationScoped
+class ScriptController {
+
+    @Inject
+    lateinit var scriptProcessor: ScriptProcessor
+
+    /**
+     * Runs given scripts
+     *
+     * @param scripts scripts
+     */
+    fun runScripts(scripts: List<MetaformScript?>?) {
+        scripts?.stream()?.forEach { script: MetaformScript? -> runScript(script) }
```

Comment:
I don't think you need java 8 streams with kotlin, just run the foreach on the list

Reply:
```suggestion
        scripts?.forEach {
            runScript(it)
        }
```
Should technically work.